crates/sui-protocol-config/src/lib.rs

```diff
@@ -1468,6 +1468,7 @@ impl ProtocolConfig {
                     cfg.feature_flags.enable_jwk_consensus_updates = true;
                     cfg.max_jwk_votes_per_validator_per_epoch = Some(240);
                     cfg.max_age_of_jwk_in_epochs = Some(1);
+                    cfg.gas_model_version = Some(7);
```

Comment:
We will need to make sure this goes out with the testnet release tomorrow, or else delay this to v26.

Reply:
The decision here depends on whether the change we're guarding is already going out with the testnet release -- I think the answer is no given how recently it landed, which means we should delay, right?

Reply:
moved to 26

Reply:
code is here at 26 however we are not bumping the protocol version in this PR.
This should simply fix the immediate problem and add the behavior with 26 which will be soon